tests/5.1/allocate/test_target_calloc.c

```diff
@@ -0,0 +1,80 @@
+//===------------------------ test_calloc_host.c --------------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// Tests the omp_calloc routine to make sure memory is properly allocated
+// on the host and is zero initialized. Allocates an array and makes sure
+// each value is set to 0 to start. Then adds i to each index and validates the
+// changes persisted and were written correctly.
+//
+//===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include "ompvv.h"
+
+#define N 1024
+
+int test_omp_target_calloc() {
+
+  int errors = 0;
+
+  #pragma omp target map(tofrom: errors) map(to: N) uses_allocators(omp_default_mem_alloc)
```

Comment:
This is not the way to create an allocator in the target region.
See: https://github.com/SOLLVE/sollve_vv/blob/master/tests/5.0/target/test_target_uses_allocators_default.c

Reply:
Will polish this up and correctly allocate this